runtime/src/bank.rs

```diff
@@ -2855,7 +2842,14 @@ impl Bank {
     }
 
     pub fn get_lamports_per_signature(&self) -> u64 {
+        solana_sdk::fee_calculator::DEFAULT_TARGET_LAMPORTS_PER_SIGNATURE / 2
```

Comment:
this would break all tests that expecting zero_fee_for_test. Perhaps can return `0` is genesis is configured for test-only, otherwise return 5000